packages/components/src/placeholder/index.js

```diff
@@ -4,19 +4,13 @@
 import classnames from 'classnames';
 import useResizeAware from 'react-resize-aware';
 
-/**
- * Internal dependencies
- */
-import Icon from '../icon';
-
 /**
  * Renders a placeholder. Normally used by blocks to render their empty state.
  *
  * @param  {Object} props The component props.
  * @return {Object}       The rendered placeholder.
  */
 function Placeholder( {
-	icon,
```

Comment:
That's something that should probably be communicated with a dev note. There's probably other code lines we can remove. (All the places where we pass an icon to the component)

Reply:
If you like, we can do this as a separate PR, perhaps that's better? In fact it might even be good to postpone doing this, and doing it together with the separate task which is to combine the title and the description into one, i.e. deprecate the description.

Reply:
Ideally, the removal of the icon prop and its usage happens at the same time. Let's postpone this to its dedicated PR then.

Reply:
Done:

<img width="724" alt="Screenshot 2020-03-03 at 09 30 10" src="https://user-images.githubusercontent.com/1204802/75756852-9f8fbf00-5d31-11ea-975b-ffe692c075ec.png">